.gitlab/pipeline/zombienet.yml

```diff
@@ -9,6 +9,7 @@
     RUN_IN_CI: "1"
     KUBERNETES_CPU_REQUEST: "512m"
     KUBERNETES_MEMORY_REQUEST: "1Gi"
+    NODE_OPTIONS: "--max_old_space_size=8192"
```

Comment:
```suggestion
    NODE_OPTIONS: "--max-old-space-size=8192"
```

Reply:
Thanks for the feedback! works both ways but I will change it to use `-` :)

```
 node -e 'console.log(v8.getHeapStatistics().heap_size_limit)'
4345298944
```

```
NODE_OPTIONS=--max_old_space_size=8192 node -e 'console.log(v8.getHeapStatistics().heap_size_limit)'
8640266240
```

```
NODE_OPTIONS=--max-old-space-size=8192 node -e 'console.log(v8.getHeapStatistics().heap_size_limit)'
8640266240
```

Reply:
Javascript, lol :P